app/views/helpers/address.scala.html

```diff
@@ -16,112 +16,115 @@
-                    <div id="textAddressData"></div>
+                    <div class="govuk-body" id="textAddressData"></div>
                 </div>
             </div>
         </div>
 
-        <div class="address--fields show-if-no-js" @if(addressData.isEmpty | selectedAddress.nonEmpty){style="display: none"} else {style="display: block;"}>
-
-            @inputText(
-                field = form(s"$fieldName.line1"),
-                '_label -> Messages(s"label.$fieldName"),
-                '_labelClass -> "form-label-bold",
-                '_help -> Html(Messages("details.edit.postcode.help")),
-                '_error -> form.error("address"),
-                'class -> "form-control",
-                'placeholder -> Messages("placeholder.address.line1"),
-                '_fieldsetClass -> "margin-bottom-0",
-                '_value -> addressData.fold("")(_.line1)
-            )
-
-            @inputText(
-                field = form(s"$fieldName.line2"),
-                '_label -> Messages(s"label.$fieldName.line2"),
-                '_labelClass -> "visuallyhidden",
-                'class -> "form-control",
-                'placeholder -> Messages("placeholder.address.line2"),
-                '_fieldsetClass -> "margin-bottom-0",
-                '_value -> addressData.fold("")(_.line2)
-            )
-
-            @inputText(
-                field = form(s"$fieldName.line3"),
-                '_label -> Messages(s"label.$fieldName.line3"),
-                '_labelClass -> "visuallyhidden",
-                'class -> "form-control",
-                'placeholder -> Messages("placeholder.address.line3"),
-                '_fieldsetClass -> "margin-bottom-0",
-                '_value -> addressData.fold("")(_.line3)
-            )
-
-            @inputText(
-                field = form(s"$fieldName.line4"),
-                '_label -> Messages(s"label.$fieldName.line4"),
-                '_labelClass -> "visuallyhidden",
-                'class -> "form-control",
-                'placeholder -> Messages("placeholder.address.line4"),
-                '_fieldsetClass -> "margin-bottom-20",
-                '_value -> addressData.fold("")(_.line4)
-            )
-
-            @inputText(
-                field = form(s"$fieldName.postcode"),
-                '_label -> Messages("label.address.postcode"),
-                '_labelClass -> "form-label-bold",
-                'class -> "form-control form-control-1-4",
-                '_error -> form.error("address.postcode"),
-                '_value -> addressData.fold("")(_.postcode)
-            )
-
-            <a href="#" class="lookupAddress hide-if-no-js">@messages("addressLookup.lookup")</a>
+<div class="address--fields show-if-no-js" @if(addressData.isEmpty | selectedAddress.nonEmpty){style="display: none"} else {style="display: block;"}>
+
+
+            @GovukInput(Input(
```

Comment:
shouldn't this be `govukInput` ? (lowercase `g`)

Reply:
@darrell-plant  no because I'm not passing in @this()  as this file is called within another template

Reply:
@zahiriqbal then you need to pass the 'govukInput' from the calling template into address. We are not allowed to use the upper-case components any longer